source/tools/testConsistency/dotnetWPF-renderer/test_app.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Microsoft Corporation. All rights reserved. 
+# Licensed under the MIT License. 
+import unittest
+import json
+from app import app
+from copy import deepcopy
+
+
+POST_URL = 'http://127.0.0.1:5000/'
```

Comment:
Ideally this would be configurable to allow for different listening ports for the server

Reply:
I modified it so that the POST_URL is simply the route instead of the URL